src/backend/src/utils/change-requests.utils.ts

```diff
@@ -49,3 +52,55 @@ export const sendSlackCRReviewedNotification = async (slackId: string, crId: num
 
   return Promise.all(msgs);
 };
+
+/**
+ *
+ * @param workPackages The Projects work Packages to be checked
+ * @param wbsElement The wbs element to check for
+ * @param timelineImpact the timeline impact of the proposed solution
+ * @param crId the change request id
+ * @param reviewer the reviewer of the change request
+ *
+ * This function will recursively update the start date of all work packages that depend on the wbs element to be finished
+ */
+export const updateDependencies = async (
+  workPackages: Prisma.Work_PackageGetPayload<typeof workPackageDependencyQueryArgs>[],
+  wbsElement: WBS_Element,
+  timelineImpact: number,
+  crId: number,
+  reviewer: User
+) => {
+  //cycle through all the work packages
```

Comment:
```suggestion
  // cycle through all the work packages
```
this bugs me. generally you should match whatever already exists in the codebase